paysafeapipaymenthubv1-apiary.apib

```diff
@@ -563,6 +577,11 @@ The overlay contains a withdraw button that creates a payment handle for the dat
                         "consumerId": "john.doe@email.com",
                         "emailSubject": "Skrill Payout",
                         "emailMessage": "Your Skrill Payout request has been processed"
+                    },
+                    "instantach": {
```

Comment:
instantach does not support any custom object attributes at this time.  These are potentially valid but is not currently implemented in APM so should be removed for now.